Nodejs/Product/Nodejs/visualstudio_nodejs_repl.js

```diff
@@ -120,8 +122,6 @@ function processRequest(command) {
                             throw err;
                         }
                     }
-                } else {
-                    obj = vm.runInContext(evalPrefix + '(' + command["code"] + ')', context);
```

Comment:
Methinks that month-ago @mjbvz was suffering from a bad case of the copy-pastes. 

Questions to ask with any introduced regression like this: why wasn't this caught sooner and how can we prevent this same mistake from happening again. Adding tests is an important step in addressing both of those questions, so I've opened #1063 to track this work.